triage/summarize/files.go

```diff
@@ -373,7 +374,7 @@ func getJSON(filepath string, v interface{}) error {
 	// Decode the JSON into the provided interface
 	err = json.Unmarshal(contents, v)
 	if err != nil {
-		return fmt.Errorf("Could not unmarshal JSON: %s", err)
+		return fmt.Errorf("Could not unmarshal JSON: %s\nTried to unmarshal the following: %#v", err, v)
```

Comment:
I think we want %w over %s. 
```
In Go 1.13, the fmt.Errorf function supports a new %w verb. When this verb is present, the error returned by fmt.Errorf will have an Unwrap method returning the argument of %w, which must be an error. In all other ways, %w is identical to %v.
```
https://blog.golang.org/go1.13-errors

Reply:
I didn't know about this. However, seeing as I've used %s throughout the entirety of this port, this might be better suited to a separate PR where all errors use %w?

Reply:
%w is only useful if you're going to unwrap the "stack" of errors, and imho is inferior to other solutions like github.com/pkg/errors